tests/test_llms_text-generation_vllm-ray.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+ip_address=$(hostname -I | awk '{print $1}')
+
+function build_docker_images() {
+    ## Build VLLM Ray docker
+    cd $WORKPATH/comps/llms/text-generation/vllm-ray/docker
+    docker build \
+        -f Dockerfile.vllmray  \
+        -t vllm_ray:habana --network=host ../../
+
+    ## Build OPEA microservice docker
+    cd $WORKPATH
+    docker build \
+        -t opea/llm-vllm-ray:latest \
```

Comment:
For test purpose, let name it as `opea/llm-vllm-ray:comps`

Reply:
updated.